src/main/scala/no/ndla/articleapi/service/search/SearchService.scala

```diff
@@ -77,15 +78,15 @@ trait SearchService {
     def matchingQuery(query: Iterable[String], withIdIn: List[Long], language: Option[String], license: Option[String], page: Option[Int], pageSize: Option[Int], sort: Sort.Value): SearchResult = {
       val searchLanguage = language.getOrElse(Language.DefaultLanguage)
 
-      val titleSearch = QueryBuilders.matchQuery(s"title.$searchLanguage", query.mkString(" ")).operator(MatchQueryBuilder.Operator.AND)
-      val contentSearch = QueryBuilders.matchQuery(s"content.$searchLanguage", query.mkString(" ")).operator(MatchQueryBuilder.Operator.AND)
-      val tagSearch = QueryBuilders.matchQuery(s"tags.$searchLanguage", query.mkString(" ")).operator(MatchQueryBuilder.Operator.AND)
+      val titleSearch = QueryBuilders.matchQuery(s"title.$searchLanguage", query.mkString(" ")).operator(Operator.AND)
+      val contentSearch = QueryBuilders.matchQuery(s"content.$searchLanguage", query.mkString(" ")).operator(Operator.AND)
+      val tagSearch = QueryBuilders.matchQuery(s"tags.$searchLanguage", query.mkString(" ")).operator(Operator.AND)
 
       val fullSearch = QueryBuilders.boolQuery()
         .must(QueryBuilders.boolQuery()
-          .should(QueryBuilders.nestedQuery("title", titleSearch))
-          .should(QueryBuilders.nestedQuery("content", contentSearch))
-          .should(QueryBuilders.nestedQuery("tags", tagSearch)))
+          .should(QueryBuilders.nestedQuery("title", titleSearch, ScoreMode.None))
```

Comment:
````
package org.apache.lucene.search.join;

/**
 * How to aggregate multiple child hit scores into a single parent score.
 */
public enum ScoreMode {

  /**
   * Do no scoring.
   */
  None,

  /**
   * Parent hit's score is the average of all child scores.
   */
  Avg,

  /**
   * Parent hit's score is the max of all child scores.
   */
  Max,

  /**
   * Parent hit's score is the sum of all child scores.
   */
  Total,

  /**
   * Parent hit's score is the min of all child scores.
   */
  Min

````
Dette kom til som et parameter krav i ES5, var tidligere default None i koden til ES2 - derfor tror/håper jeg det er dette som blir riktig for oss nå; også siden metodesignaturen virker å være endret for å tvinge frem bevist valg her istedet for å lene seg på hemmelig default (?).